pages/bib/[id].tsx

```diff
@@ -49,8 +109,50 @@ export default function BibPage({
       <Layout isAuthenticated={isAuthenticated} activePage="bib">
         <Heading level="h2">{bib.title}</Heading>
         <BibDetails key="top-details" details={topDetails} />
-        {bib.itemTableData ? (
-          <ItemTable itemTableData={bib.itemTableData} />
+        {bib.showItemTable ? (
+          <Box id="item-table">
+            <Heading
+              data-testid="item-table-heading"
+              level="h3"
+              size="heading4"
+              mb={{ base: "s", md: "m" }}
+            >
+              Items in the library and off-site
+            </Heading>
+            {itemsLoading ? (
+              <SkeletonLoader showImage={false} />
+            ) : itemFetchError ? (
+              <Notification
```

Comment:
Ah, use the `Banner` component for notifications that are local to the page. https://nypl.github.io/nypl-design-system/reservoir/v3/?path=/docs/components-feedback-banner--docs -- use the "negative" variant.